src/main/resources/application.conf

```diff
@@ -0,0 +1,2 @@
+  json = "//file.json"
```

Comment:
strange path. About naming, let it be channelResultPath, or channelSavePath. Your naming isn't about real purpose of the variable - fix it.